docs/rc.xml.all

```diff
@@ -77,7 +77,20 @@
         <field content="output" width="9%" />
         <field content="identifier" width="30%" />
         <field content="title" width="50%" />
-       </fields>  
+      </fields>  
+    </windowSwitcher>
+
+    A new OSD format - custom
+    It effectively replaces all the above "fields" with one line, using a
+    printf style format. The example below would print "foobar",then type of
```

Comment:
Presumably this (`all the above "fields" with one line`) is one way of using it.
Other uses-cases could be to have multiple `custom` fields for more flexibility.
That description ought to reflect this.

Reply:
Done